packages/roosterjs-editor-plugins/lib/plugins/ContentEdit/features/listFeatures.ts

```diff
@@ -479,7 +479,7 @@ function shouldTriggerList(
     const traverser = editor.getBlockTraverser();
     const text =
         traverser && traverser.currentBlockElement
-            ? traverser.currentBlockElement.getTextContent()
+            ? traverser.currentBlockElement.getTextContent().slice(0, textBeforeCursor.length)
```

Comment:
I remember we were using PositionContentSearcher to help find text before cursor before. Any reason we don't use it now?

Reply:
We still use PositionContentSearcher to find the text before cursor, the code I added is to check if the trigger character is the same text on the begging of the paragraph. The aim of this code is to avoid the list being trigger in unexpected positions, since we have more characters that triggers lists.